apps/api/src/main.ts

```diff
@@ -18,7 +18,7 @@ async function bootstrap() {
   config.configureSwagger(app)
   app.use(cookieParser())
   try {
-    await app.listen(config.port)
+    await app.listen(config.port, '0.0.0.0')
```

Comment:
This in order to make nodejs handles all the request as IPV4, because by default it uses IPV6.
We can remove it, but then we need to defined how are we are going to handle IPV6 vs IPV4.